csi/deployment.go

```diff
@@ -387,6 +391,20 @@ func NewPluginDeployment(namespace, serviceAccount, nodeDriverRegistrarImage, ma
 								},
 							},
 						},
+						{
+							Name:            "longhorn-liveness-probe",
+							ImagePullPolicy: imagePullPolicy,
+							Image:           livenessProbeImage,
+							Args: []string{
+								fmt.Sprintf("--csi-address=%s", GetInContainerCSISocketFilePath()),
```

Comment:
No verbose? Just have a try to see what level is much suitable. 

ref: https://github.com/kubernetes-csi/livenessprobe/blob/master/deployment/kubernetes/livenessprobe-sidecar.yaml#L51

Reply:
Yes, we should set verbose, I think the default is set to `v=2`. 
`v=5` output the [probe request](https://github.com/kubernetes-csi/livenessprobe/blob/master/cmd/livenessprobe/main.go#L69) and the [connections](https://github.com/kubernetes-csi/livenessprobe/blob/master/vendor/github.com/kubernetes-csi/csi-lib-utils/connection/connection.go#L183-L187), so set `v=4` should be at the debug level and looks right to me.

Reply:
Ok, I think v=4 is still too much. Need to make it less verbose.
```
[longhorn-csi-plugin-hf5vb longhorn-csi-plugin] time="2022-11-29T03:27:43Z" level=info msg="Probe: rsp: {}"
[longhorn-csi-plugin-rsvwr longhorn-csi-plugin] time="2022-11-29T03:27:43Z" level=info msg="Probe: req: {}"
```